WebService/ComparisonEngineUI/ComparisonEngineUI/Data/RestService.cs

```diff
@@ -0,0 +1,239 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WebService.Base_Classes;
+
+namespace ComparisonEngineUI.Data
+{
+    class RestService
+    {
+        HttpClient client;
+
+        public RestService()
+        {
+            client = new HttpClient();
+        }
+
+        public async Task<List<Bar>> GetBars()
+        {
+            Uri uri = new Uri(Constants.BarsUrl);
+            try
+            {
+                HttpResponseMessage response = await client.GetAsync(uri);
+
+                if(!response.IsSuccessStatusCode)
+                {
+                    return null;
+                }
+
+                string content = await response.Content.ReadAsStringAsync();
+                return JsonConvert.DeserializeObject<List<Bar>>(content);
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
+
+        public async Task<List<Drink>> GetDrinks()
+        {
+            Uri uri = new Uri(Constants.DrinksUrl);
+            try
+            {
+                HttpResponseMessage response = await client.GetAsync(uri);
+
+                if (!response.IsSuccessStatusCode)
+                {
+                    return null;
+                }
+
+                string content = await response.Content.ReadAsStringAsync();
+                return JsonConvert.DeserializeObject<List<Drink>>(content);
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
+
+        public async Task<List<SpecificPrice>> GetSpecificPrices()
+        {
+            Uri uri = new Uri(Constants.SpecificPricesUrl);
+            try
+            {
+                HttpResponseMessage response = await client.GetAsync(uri);
+
+                if (!response.IsSuccessStatusCode)
+                {
+                    return null;
+                }
+
+                string content = await response.Content.ReadAsStringAsync();
+                return JsonConvert.DeserializeObject<List<SpecificPrice>>(content);
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
```

Comment:
These methods, as well as some others, could probably be merged into one by passing what type of data we want to get back to a method, and then performing "if" checks in the places where the code is different